### PR TITLE
r-fastcluster

### PR DESCRIPTION
**Pull request by jsignell**

### CODE REVIEW

File: recipes/r-fastcluster/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
```
Comment (bgruening) [inline recipes/r-fastcluster/meta.yaml]: libgcc

File: recipes/r-fastcluster/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+test:
+  commands:
+    - $R -e "library('fastcluster')"  # [not win]
+    - "\"%R%\" -e \"library('fastcluster')\""  # [win]
+
+about:
+  home: http://danifold.net/fastcluster.html
```
Comment (bgruening) [inline recipes/r-fastcluster/meta.yaml]: can you add `license_file: LICENSE` here

File: recipes/r-fastcluster/meta.yaml (hunk 1 above)
Comment (jsignell) [inline recipes/r-fastcluster/meta.yaml]: @bruening, thanks for all the reviews! Do you just mean that I need to add the line 
```yaml
  - libgcc  # [not win]
```
on line 34?

Comment (bgruening) [inline recipes/r-fastcluster/meta.yaml]: yes :)

Comment (jsignell) [inline recipes/r-fastcluster/meta.yaml]: nevermind. I just saw the discussion here: https://github.com/conda-forge/staged-recipes/pull/3302#pullrequestreview-50206892

**Pull request merged**